### ISSUE TITLE
Using existing Schema classes as a ref for Example classes

### ISSUE THREAD

**Issue by shahrukh4** [zircote/swagger-php#1535]
I am not sure if this feature is supported by this package or not. If it is, can anyone help me with this, as I am facing a little issue here.
If the below will compile to the response like follows, so why it can't be used for Examples classes and taking it as as value. I want it to take the value from the existing schema, so I don't have to give the hardcoded value explicitly. If it is not possible, can anyone create a PR or something against it?

OpenAPI code block

```
use OpenApi\Attributes\Property;
use OpenApi\Attributes\Schema;
use OpenApi\Attributes\Examples;

#[Schema()]
class SchemaYourSchemaNameClass
{
    #[Property()]
    #[Examples()]
    public int $productId;

    #[Property()]
    #[Examples()]
    public string $name;
}
```

Now you can use this for your response.

```
use OpenApi\Attributes\Schema;
use OpenApi\Attributes\JsonContent;

#[Response(
    response: 'YourResponse',
    description: 'description',
    content: new JsonContent(
       anyOf: [
            new Schema(ref: SchemaYourSchemaNameClass::class),
        ],
        examples: [
            new Examples(
                example: 'LineChartExample',
                ref: '#/components/examples/SchemaYourSchemaNameClass',
            ),
        ],
    )
)]
```

Response it is generating

```
{
    date: 'string',
    value: 0,
}
```

**Comment by DerManoMann** [comment]
The spec (https://spec.openapis.org/oas/v3.1.0.html#example-object) defines `value` as `Embedded literal example`. That probably excludes using it the way you want to.

Thinking backwards - do you have some example YAML that would do what you want in a tool like swagger-ui?

**Comment by shahrukh4** [comment]
@DerManoMann Thanks for responding quickly on this, I went through documentations and what you are saying is correct. Following are the way, I achieved what I exactly wanted

- I was needed to pass the already built schema class to the `Examples::class` ref and I want it to use as a value array, so I've achieved it by creating a `CustomExamples::class` which can take a `ref` parameter and get the exact structure out of it and assign it to the value.
- Closing this ticket, as it is no longer needed. Thanks @all .
